src/toil/jobStores/googleJobStore.py

```diff
@@ -24,6 +24,7 @@
 import logging
 import time
 import os
+import os.path
```

Comment:
Nit: `import os.path` and `import os`.

Reply:
I totally thought you needed both. Apparently you don't.